axelor-account/src/main/java/com/axelor/apps/account/service/analytic/AnalyticAccountServiceImpl.java

```diff
@@ -119,14 +118,17 @@ public String getAnalyticAccountDomain(
       } else {
         domain +=
             "in ("
-                + accountConfigRepository.findByCompany(analyticDistributionTemplate.getCompany())
-                    .getAnalyticAxisByCompanyList().stream()
-                    .map(it -> it.getAnalyticAxis().getId().toString())
-                    .collect(Collectors.toList())
+                + Joiner.on(",")
+                    .join(
+                        accountConfigRepository
+                            .findByCompany(analyticDistributionTemplate.getCompany())
+                            .getAnalyticAxisByCompanyList().stream()
+                            .map(it -> it.getAnalyticAxis().getId().toString())
+                            .collect(Collectors.toList()))
```

Comment:
Please improve this part because if AnalyticAxisByCompanyList is empty, we'll get an exception.
Add a 0 by default to always have something in the `in(...)`